server/bin/pbench-backup-tarballs.py

```diff
@@ -212,6 +213,7 @@ def backup_to_local(lb_obj, logger, controller_path, controller, tb,
             else:
                 tar_done = True
 
+        logger.debug("End local backup of {}.".format(tar))
         if md5_done and tar_done:
             logger.info(
                 "Locally backed-up sucessfully: {tarball}".format(tarball=os.path.join(controller, resultname)))
```

Comment:
Can you also change this wording to something along the lines of what you added in the other log statements?

Perhaps: `"Local backup of {tarball} successful".format(tarball=...)`